### DISCUSSION TITLE
Welcome to SimplifyReport_Backend Discussions!

### DISCUSSION THREAD

**Discussion by vatsalkesarwani12** [AbsurdNerd/SimplifyReport_Backend#4] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anshsrtv** [comment]
Hello everyone 👋
I am Ansh Srivastava, a Pre-Final year undergraduate at NIT Raipur. 
I am here to learn from my peers and mentors. 🚀 
I proposed to add SMS and Email alerts in this application. Also, a natural disaster API can be introduced alongside other APIs for areas prone to such disasters.

**Reply by diyajaiswal11** [reply]
Hey @anshsrtv , Glad to have you here!! I'm assigning **SMS and Email alerts** issue to you currently. You can start working on it and can coordinate and discuss anything with your fellow participants here and us in the Q&A section.

**Comment by InfiniteVerma** [comment]
Hey everyone. 
I am Anant Verma, 3rd year student. 
My proposal contained three parts: 1) Add authentication, 2) Add functionality for authorities to view and del alerts, 3) document the API.

**Reply by diyajaiswal11** [reply]
Hey @InfiniteVerma  , Glad to have you here!! I'm assigning **Authentication** issue to you currently. You can start working on it and can coordinate and discuss anything with your fellow participants here and us in the Q&A section.

**Comment by vramanrs** [comment]
Hello everyone, I am Venkata Raman. I am in my 3rd year. I proposed to add social login. I want to learn from my mentors and peers.

**Reply by diyajaiswal11** [reply]
Hey @rv-raman , Glad to have you here!! I'm assigning **Social Authentication** issue to you currently. You can start working on it and can coordinate and discuss anything with your fellow participants here and us in the Q&A section.

**Comment by diyajaiswal11** [comment]
Guys @anshsrtv @InfiniteVerma @rv-raman , please comment on your respective issues, so that I can assign it to you.
